chia/cmds/param_types.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from decimal import Decimal, InvalidOperation
+from typing import Any, Callable, ClassVar, Optional, Union
+
+import click
+
+from chia.cmds.units import units
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.bech32m import bech32_decode, decode_puzzle_hash
+from chia.util.config import selected_network_address_prefix
+from chia.util.ints import uint64
+from chia.wallet.util.address_type import AddressType
+
+one_decimal_mojo = Decimal("0.000000000001")
```

Comment:
Just helps out with readability.
```suggestion
one_decimal_mojo = Decimal("0.000_000_000_001")
```

or...

```python-console
>>> Decimal("0.000_000_000_001") == Decimal("1e-12")
True
```

```suggestion
one_decimal_mojo = Decimal("1e-12")
```